### ISSUE TITLE
Feature Request: Start composin post with mention of user from users profile

### ISSUE THREAD

**Issue by ben-utzer** [friendica/friendica#8794]
I would like to have button added to the users profile and to the contact overview that allows me to click on it and start /compose with a mention of that user. 

Like Twitter got this, when you're on a users profile you can there start a new tweet to the user, so the compose field of the form is prefilled with and ```@mention```.

**Comment by walstonjd** [comment]
I'd like to have a crack at this. Can I get some info on getting started?

**Comment by MrPetovan** [comment]
Yes. The Compose controller is in `src/Module/Compose.php`. This is where you’re supposed to collect GET parameters and change the initial value of the body tectarea. Let me know if you need more info.

**Comment by walstonjd** [comment]
> Yes. The Compose controller is in `src/Module/Compose.php`. This is where you’re supposed to collect GET parameters and change the initial value of the body tectarea. Let me know if you need more info.

I don't see `src/Module/Compose.php` in the Module folder, is that the correct location?

**Comment by AlfredSK** [comment]
It's `src/Module/Item/Compose.php`, I guess.

**Comment by MrPetovan** [comment]
Sorry, I answered from the top of my head on my phone.

**Comment by walstonjd** [comment]
Thanks, I found the file.

**Comment by walstonjd** [comment]
would something like this be appropriate? I made a Mention area in the dev tools.
![Screenshot from 2020-07-25 23-18-43](https://user-images.githubusercontent.com/59633869/88470683-6701a500-cecd-11ea-9a2f-19f32a71915c.png)

**Comment by MrPetovan** [comment]
I'm on board with the "Mention" button, not with the second line with the actual mention in the vertical card (vcard).

However I believe this is the vier theme and the Compose page is only accessible from the frio theme, so the link should only be added to frio-specific templates. These are located in the `view/theme/frio` folder.

**Comment by walstonjd** [comment]
Thanks for clearing that up. Could you give me some more guidance on modifying the GET parameters in Compose.php

**Comment by MrPetovan** [comment]
In PHP, when you add a query string parameter to a page URL, its key and value end up in the superglobal `$_GET` array. For example, `/compose?mention=joe@example.com` would give the following `$_GET` array:
```php
array (1) (
    "mention" => "joe@example.com"
)
```
And you can retrieve the value in `Compose.php` with `$_GET['mention']`.

**Comment by walstonjd** [comment]
Where should I add `$_GET['mention']` in `Compose.php`?

**Comment by MrPetovan** [comment]
Pardon me for asking, but do you have any prior knowledge in PHP programming? If not, it will make this process painstaking for both of us. I appreciate your willingness to contribute but it wouldn't be worth it overall if I spend more time helping you than adding the feature myself.

**Comment by walstonjd** [comment]
I have minimal experience with PHP. I agree that this one is over my head so I'll back off of this issue.

**Comment by MrPetovan** [comment]
Thank you kindly.

**Comment by marcusxss** [comment]
I think this might be resolved, so it can be closed, or?
